### DISCUSSION TITLE
Move sections between views

### DISCUSSION THREAD

**Discussion by tomlut** [home-assistant/frontend#22713] [Dashboard]
It would be very handy to be able to move sections between views. Like cards can be moved to other views and even other dashboards. 

Obviously, only to other views of type "Section".

**Comment by KNXBroker** [comment]
Identified the same, also duplication should be offered not only moving.

**Comment by tomlut** [comment]
This would have been super handy when replacing most of my custom cards with sections and tile cards recently.
